docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -179,11 +167,10 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 --request GET 'https://api.us-west-1.saucelabs.com/api-testing/rest/v4/<hookId>' | json_pp
 ```
 
-</TabItem>
 
 If you're using Webpack as your project bundler, you can use `@backtrace/webpack-plugin` to automate working with sourcemaps.
 
-### Step 1: Enable Source Maps for Your Application
+#### Step 1: Enable Source Maps for Your Application
```

Comment:
I made these instructions indent level 4 so they don't show up in the sidebar.

Reply:
Actually, this is not correct since the levels should go in order:  Project Bundlers is level 2, so "steps" should have level 3. If you want to remove them from the sidebar, we can remove them as heading

Reply:
Even better!